cli/src/output/table.rs

```
@@ -57,7 +57,7 @@ pub fn fmt_u64_separate_with_commas(o: &u64) -> String {
 pub fn fmt_option_milliseconds(o: &Option<i64>) -> String {
     match o {
         Some(s) => {
-            let duration = Duration::milliseconds(*s);
+            let duration = Duration::try_milliseconds(*s).unwrap();
```

Anyway to avoid unwrap?

---

Ya we could, but this is just for overflow checking, so i think instead of using a default value, just make it panic. So we know the overflow happen.

---

because overflow is very hard to notice

---

Cool